### ISSUE TITLE
files for automatic sidecar injection missing from istio 0.5.1 distribution

### ISSUE THREAD

**Issue by sdake** [istio/istio#3737]
The file `/install/kubernetes/webhook-create-signed-cert.sh` is missing.

This may be fixed in the upcoming 0.6.0, just wanted to make sure it was.  I am not sure what generates the tarball:

```
sdake@falkor-07:~/051/istio-0.5.1$ !find
find . -name \*sh -ls
 40635429      4 -rwxr-xr-x   1 sdake    sdake        2923 Feb 12 16:11 ./tools/deb/istio-start.sh
 40635435      4 -rwxr-xr-x   1 sdake    sdake        1355 Feb 12 16:11 ./tools/deb/postinst.sh
 40635431      4 -rwxr-xr-x   1 sdake    sdake        1584 Feb 12 16:11 ./tools/deb/istio-ca.sh
 40635426      4 -rwxr-xr-x   1 sdake    sdake         793 Feb 12 16:11 ./tools/deb/deb_test.sh
 40635433      8 -rwxr-xr-x   1 sdake    sdake        6639 Feb 12 16:11 ./tools/deb/istio-iptables.sh
 40635424     12 -rwxr-xr-x   1 sdake    sdake        9252 Feb 12 16:11 ./tools/setup_perf_cluster.sh
 40635422      4 -rw-r--r--   1 sdake    sdake        2806 Feb 12 16:11 ./tools/vagrant/provision-vagrant.sh
 40370620     12 -rwxr-xr-x   1 sdake    sdake        8407 Feb 12 16:11 ./install/tools/setupMeshEx.sh
 40370619      4 -rwxr-xr-x   1 sdake    sdake        3597 Feb 12 16:11 ./install/tools/setupIstioVM.sh
 40763783      4 -rwxr-xr-x   1 sdake    sdake        1476 Feb 12 16:11 ./samples/bookinfo/kube/cleanup.sh
 40763798      4 -rwxr-xr-x   1 sdake    sdake        1541 Feb 12 16:11 ./samples/bookinfo/consul/cleanup.sh
 40763810      4 -rwxr-xr-x   1 sdake    sdake        1541 Feb 12 16:11 ./samples/bookinfo/eureka/cleanup.sh
```

cc @ayj

**Comment by ayj** [comment]
This should be fixed in 0.6.0. The troubleshooting page has a section on this issue (see https://istio.io/help/troubleshooting.html#kubernetes-webhook-setup-script-files-are-missing-from-05-release-package)

**Comment by sdake** [comment]
thanks @ayj.
